plugins/kotlin/compiler-plugins/parcelize/tests/test/org/jetbrains/kotlin/pacelize/ide/test/parcelizeTestUtil.kt

```diff
@@ -12,7 +12,7 @@ import java.io.File
 
 fun addParcelizeLibraries(module: Module) {
     ConfigLibraryUtil.addLibrary(module, "androidJar") {
-        addRoot(File(PathManager.getHomePath(), "community/android/android/testData/android.jar"), OrderRootType.CLASSES)
+        addRoot(File(PathManager.getHomePath(), "android/android/testData/android.jar"), OrderRootType.CLASSES)
```

Comment:
It's not wrong. Inside JetBrains we have big monorepo and `intellij-community` you observe on GitHub is subdirectory in this monorepo.

I propose another fix, change `PathManager.getHomePath()` to `PathManager.getCommunityHomePath()`

Reply:
Thanks for the information, that makes a lot of sense! I've updated the code to use `getCommunityHomePath`.